docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -204,7 +204,7 @@ Prepare `fabric-cli` for configuration suitably as follows.
   * Update value for `relayEndpoint` for `Corda_Network` as `relay-corda:9081`.
   * Update value for `relayEndpoint` for `Corda_Network2` as `relay-corda2:9082`.
   * Update value for `partyEndPoint` for `Corda_Network` as `corda_partya_1:10003`.
-  * Update value for `partyEndPoint` for `Corda_Network2` as `corda_network2_partya_1:10003`.
+  * Update value for `partyEndPoint` for `Corda_Network2` as `corda_network2_partya_1:30003`.
```

Comment:
Can we revert this? Previous one was correct, this is wrong. (We are talking about dockerized here, not localhost)